### ISSUE TITLE
A question about using val.py to verify the trained model

### ISSUE THREAD

**Issue by Jiangdiwen** [JDSobek/MedYOLO#24]
Hi，I simply trained a model using 40 examples of data. After the model training was completed, the accuracy index of the validation set was printed, as shown in the figure below.
![85dcf81bbe7ce30fe8b3ee6a12d3c14](https://github.com/user-attachments/assets/55d7bf1c-035c-4bab-85de-6fef29e31255)

I had a test set of 10 cases (with labels) and wanted to see how the model performed, so I tried to use `val.py` to test, hoping to get an accuracy indicator.But no matter what model weights I use, I end up with the same result...... The effect is too good to be true:
![image](https://github.com/user-attachments/assets/f6d4e4df-32ad-47ae-8778-c4552f473c69)

Then I set a breakpoint in line 74 Func `process_batch `to see what the model predicted, and found that every decimal point was the same as the label value, but I didn't quite understand what was going on.
![image](https://github.com/user-attachments/assets/0cef42df-a18d-496e-b54a-2545d9e4c130)

This is my parameter setting. I don’t know if it has anything to do with the fact that I have changed something.Do you know why?
![image](https://github.com/user-attachments/assets/3e8b2b07-bda7-4c48-8f29-68911649647d)

Thank you for every answer ！ lol

**Comment by JDSobek** [comment]
Somehow I've never used `save-hybrid` before... Interesting. Can you run `val.py` without save-hybrid on?

**Comment by Jiangdiwen** [comment]
You're right. When I disabled that option, output was fine !
